emailsig/templates/email_signature.html

```diff
@@ -1,22 +1,69 @@
-    <img src="https://twig.s3.amazonaws.com/images/email/signature/twig_logo.jpg" alt="Twig World Ltd" border="0">
-</a>
-<a style="text-decoration:none; line-height:8px;" href="http://www.tigtagworld.com">
-    <img src="https://twig.s3.amazonaws.com/images/email/signature/tigtag_logo.jpg" alt="Tigtag" border="0">
-</a>
-<p style="font-family: Helvetica, Arial, sans-serif; font-size: 12px; line-height:24px; color: rgb(102, 102, 102)">
-    <span style="font-weight: bold;">{{ request.session.full_name }}</span> | {{ request.session.job_title }}
-<br>
+{% extends "base.html" %} {% block content %}
+            <table style="font-family: Helvetica, Arial;">
+                <tr>
+                    <td>
+                        <table cellpadding="10" width="158" height="120">
+                            <tr>
+                                <td colspan="5" style="border-right: 1px solid !important">
+                                    <a style="text-decoration:none; line-height:8px;" href="https://www.twigeducation.com/"><img src="https://res.cloudinary.com/dodge7ws8/image/upload/c_scale,f_png,h_56/v1517403916/email-signature/twig_education.svg" alt="Twig Education"/></a>
+                                </td>
+                            </tr>
+                            <tr height="40" colspan="5">
+                                <td colspan="5" style="border-right: 1px solid !important"></td>
+                            </tr>
+                            <tr>
+                                <td colspan="3" width="35" />
+                                <td colspan="2" style="border-right: 1px solid !important">
+                                    <a href="https://www.facebook.com/TwigWorld/"><img src="https://res.cloudinary.com/dodge7ws8/image/upload/c_scale,f_png,h_22/v1517315583/email-signature/facebook_logo.svg" alt="Facebook"/></a>
+                                    <a href="https://twitter.com/TwigWorld"><img src="https://res.cloudinary.com/dodge7ws8/image/upload/c_scale,f_png,h_22/v1517315583/email-signature/twitter_logo.svg" alt="Twitter"/></a>
+									<a href="https://www.linkedin.com/company/1342611/"><img src="https://res.cloudinary.com/dodge7ws8/image/upload/c_scale,f_png,h_22/v1517315583/email-signature/linkedin_logo.svg" alt="LinkedIn"/></a>
+                                    <a href="https://www.instagram.com/twig.world/"><img src="https://res.cloudinary.com/dodge7ws8/image/upload/c_scale,f_png,h_22/v1517315583/email-signature/instagram_logo.svg" alt="Instagram"/></a>
+                                </td>
+                            </tr>
+                        </table>
+                    </td>
+                    <td width="20"></td>
+                    <td>
+                        <table height="120">
+                            <tr style="font-size: 18px; color: #5C5C5C; font-weight: bold; height:60!important; vertical-align: top;">
+                                <td class="name">
```

Comment:
Classes should be avoided on html email as many clients don't use them. Styles should be inline.

Reply:
@adamoliver thats fine the class there doesnt add any styling it should be removed